src/colab/Run_SKAI_Colab_Pipeline.ipynb

```diff
@@ -337,11 +347,34 @@
         "#@title Generate Examples\n",
         "#@markdown First, generate the building images. This task should take about 30-45 minutes.\n",
         "\n",
+        "#@markdown Select if you want to generate your example from a pre-labeled file or generate unlabeled examples.\n",
+        "GENERATING_JOB = \"unlabeled\" #@param [\"pre_labeled\",\"unlabeled\"]\n",
+        "\n",
         "## COMMAND RUN\n",
         "timestamp = datetime.datetime.now().strftime('%Y%m%d%H%M%S')\n",
         "GCP_DATASET_NAME = f\"{Author}_example_{timestamp}_{PROJECT_DIRECTORY}\".replace(\"_\",\"-\").lower()\n",
         "\n",
-        "generate_examples_args = {\n",
+        "if GENERATING_JOB == \"pre_labeled\":\n",
+        "  generate_examples_args = {\n",
+        "    'cloud_project': GCP_PROJECT,\n",
+        "    'cloud_region': GCP_LOCATION,\n",
+        "    'dataset_name' : GCP_DATASET_NAME,\n",
+        "    'before_image_patterns': pathgcp_imagebefore,\n",
+        "    'after_image_patterns': pathgcp_imageafter,\n",
+        "    'aoi_path': pathgcp_aoi,\n",
+        "    'output_dir': f'gs://{pathgcp_outputdir}',\n",
+        "    'buildings_method': 'none',\n",
+        "    'worker_service_account': service_account,\n",
+        "    'earth_engine_service_account' : service_account,\n",
+        "    'earth_engine_private_key' : pathsys_credentials,\n",
+        "    'use_dataflow': 'true',\n",
+        "    'labels_file': pathgcp_filelabeled,\n",
+        "    'label_property': pathgcp_keylabeled,\n",
+        "    'labels_to_classes': pathgcp_mappinglabeled\n",
+        "    }\n",
+        "\n",
+        "elif GENERATING_JOB == \"unlabeled\":\n",
```

Comment:
Are there any other cases of GENERATING_JOB besides "labeled" or "unlabeled"?
If not, how about changing this to an else instead of an elif? 
If we want to make sure users choose either "labeled" or "unlabeled," how about adding a check at the beginning (e.g. if GENERATING_JOB not in ["labeled", "unlabeled"]: raise ValueError).
Unresolved

Reply:
Thanks, agree with the change for the if else. Regarding the check, it's less needed as the user has by default the choice between only the two values in the form list `#@param ["labeled","unlabeled"]` of the colab notebook